tox.ini

```diff
@@ -34,14 +34,33 @@ commands =
     lint: flake8
     lint: yamllint -s test/ molecule/
 
+[testenv:lint]
+deps = 
+    -rlint-requirements.txt
+extras = []
+skip_install = true
+tags =
+    lint
+usedevelop = false
+
 [testenv:format]
 commands =
     yapf -i -r molecule/ test/
-
+deps = yapf>=0.25.0,<2
+extras = []
+skip_install = true
+tags =
+    format
+usedevelop = false
 [testenv:format-check]
 commands =
     yapf -d -r molecule/ test/
-
+deps = yapf>=0.25.0,<2
```

Comment:
AFAIR the syntax for reusing is `deps = {[format]deps}`. Plz check...

Reply:
I don't understand?

Reply:
i eventually understood !
done